doc/lua_api.txt

```diff
@@ -1189,7 +1189,7 @@ clients and handled by many parts of the engine.
 A 'mapchunk' (sometimes abbreviated to 'chunk') is usually 5x5x5 mapblocks
 (80x80x80 nodes) and is the volume of world generated in one operation by
 map generation.
```

Comment:
`... generated in one pass by the mapgen.`

(Probably relevant, as this line gets touched as well.)

Reply:
That's ok but i think 'operation' is clearer.

Reply:
Sorry for being so vague, I was mainly suggesting to change "map generation" to "mapgen" :)

Reply:
Ahh. I have been wondering about that, i was thinking 'map generation' is clearer than 'mapgen', but 'the mapgen' refers to the core mapgen being used, which may be clearer.

Reply:
Done.